### DISCUSSION TITLE
Standardizing UniProt information

### DISCUSSION THREAD

**Discussion by sr320** [RobertsLab/resources#1277] [Q&A] [answered]
Thinking beyond the blast table is there a programatic way to capture the information in this screenshot, AND more importantly to we want to standardize, discover, explore specific parameters?
<img width="1544" alt="reviewed_yes_in_UniProtKB" src="https://user-images.githubusercontent.com/4442039/131743893-ce9c4fb8-ebca-4b14-b254-c00d3ebf4791.png">

**Comment by kubu4** [comment]
Yes. The programmatic batch retrieval grabs EVERYTHING (as if you had checked all the boxes).

**Reply by sr320** [reply]
How do I programmatically retrieve via command-line? Do you have an example?

**Comment by kubu4** [comment, marked as answer]
See `In [13]` of this notebook:

https://nbviewer.jupyter.org/github/RobertsLab/code/blob/master/notebooks/sam/20210601_ssal_gff-annotations.ipynb

You'll need to modify the input accession type in the Perl script to match whatever you're using. There's a link in the notebook to the SwissProt site and that should have mapping parameters to show you what you need as input/output accessions.
